.gitignore

```diff
@@ -29,4 +29,5 @@ node_modules
 package-lock.json
 .cache
 include/ccf/version.h
-python/version.py
\ No newline at end of file
+python/version.py
+tests/compatibility_report.json
```

Comment:
Could that be made to go under build/workspace, like other test output?

Reply:
Yep, good idea!